package.json

```diff
@@ -841,13 +841,13 @@
                     "command": "jupyter.notebookeditor.runallcellsabove",
                     "title": "%DataScience.runAbove%",
                     "group": "inline/cell@0",
-                    "when": "notebookViewType == 'jupyter-notebookxxx' && jupyter.isnotebooktrusted"
+                    "when": "notebookViewType == 'jupyter-notebook' && jupyter.isnotebooktrusted"
```

Comment:
We are already using the controller instead of the command to run above and below (https://github.com/microsoft/vscode-jupyter/blob/e2207488952e8a71693d044a811712fd760c66e5/src/client/datascience/notebook/notebookEditor.ts#L319).
This should be the only change needed

Reply:
There was a reason we dropped this. Wasn't the existing implementation contributing to https://github.com/microsoft/vscode-jupyter/issues/5737?

Reply:
Aaah yes, you are right, i guess we either ship this with the bug or don't ship it and remove the functionality. which ones the less evil...
Something to be discussed. (to me its obvious, the Push API broken this like plenty of other things 😢 )

Reply:
Yea. Don recommended using the controller, I don't know if that would regress to the original issue.
@DonJayamanne what do you think?